packages/babel-parser/src/parser/location.js

```diff
@@ -35,8 +35,7 @@ export const Errors = Object.freeze({
   ConstructorClassField: "Classes may not have a field named 'constructor'",
   ConstructorClassPrivateField:
     "Classes may not have a private field named '#constructor'",
-  // todo: rephrase to get/set accessor
-  ConstructorIsAccessor: "Constructor can't have get/set modifier",
+  ConstructorIsAccessor: "Constructor can't have get/set accessor",
```

Comment:
```suggestion
  ConstructorIsAccessor: "Constructor can't be a get/set accessor",
```

Reply:
Howabout

```suggestion
  ConstructorIsAccessor: "Constructor cannot have a get/set accessor",
```

Reply:
In
```js
class A {
  get foo() {}
}
```
I think that `foo` _is_ the accessor, not that it _has_ the accessor.